docs/extending_wsimod.md

```diff
@@ -0,0 +1,95 @@
+# Extending and customising WSIMOD functionality
+
+WSIMOD has many features built-in, but undoubtedly, to model real life scenarios, you will need to customise it to your specific needs.
+
+WSIMOD offers 4 ways of doing this customisation of increased complexity - and flexibility: custom orchestration, overrides, patches and custom classes.
+
+## Custom orchestration
+
+WSIMOD allows to define the order in which the sequence of actions for each node must take place. This orchestration can be customised within the `orchestration` section of the config file. For example, the following orchestration will run the simulation with two actions: first it will run the `infiltrate` method of the `Groundwater` node, and then the `make_discharge` method of the `Sewer` node.
+
+```yaml
+orchestration:
+- Groundwater: infiltrate
+- Sewer: make_discharge
+```
+
+If no `orchestration` is provided, then a default sequence is used. Check the [Orchestration section](./orchestration.md) for more details on how to customise this.
```

Comment:
```suggestion
If no `orchestration` is provided, then a default sequence is used. Check the [Orchestration demonstration](./../demo/scripts/oxford_demo/#orchestration) for more details on how to customise this. Note, that if you change your orchestration *at all* then you will have to provide a full orchestration for all `type_: functions` that need calling (even ones that are in the default orchestration, which you can find in [`Model.__init__`](reference-model.md#Model.__init__)).
```

Reply:
I think for now we won't bother with a devoted orchestration guide as there is already quite a lot of material here and in the paper (both of which are now linked).

Not sure if my hyperlink there is correct - I think perhaps the `mkdocs-jupyter` notebooks are not automatically picked up in the same way as markdown guides? Maybe this explains why I had a lot of the relative references...